ktor-server/ktor-server-plugins/ktor-server-webjars/jvm/src/io/ktor/server/webjars/Webjars.kt

```diff
@@ -1,126 +1,95 @@
-         * Path prefix at which the installed plugin responds.
-         */
-        public var path: String = "/webjars/"
-            set(value) {
-                field = buildString(value.length + 2) {
-                    if (!value.startsWith('/')) {
-                        append('/')
-                    }
-                    append(value)
-                    if (!endsWith('/')) {
-                        append('/')
-                    }
+    public var path: String = "/webjars/"
+        set(value) {
+            field = buildString(value.length + 2) {
+                if (!value.startsWith('/')) {
+                    append('/')
+                }
+                append(value)
+                if (!endsWith('/')) {
+                    append('/')
                 }
-            }
-
-        /**
-         * Makes no effect. Will be dropped in future releases.
-         */
-        @Suppress("unused")
-        @Deprecated("This is no longer used and will be dropped in future releases.", level = DeprecationLevel.ERROR)
-        public var zone: ZoneId = ZoneId.systemDefault()
-    }
-
-    private suspend fun intercept(context: PipelineContext<Unit, ApplicationCall>) {
-        val fullPath = context.call.request.path()
-        if (fullPath.startsWith(webjarsPrefix) &&
-            context.call.request.httpMethod == HttpMethod.Get &&
-            fullPath.last() != '/'
-        ) {
-            val resourcePath = fullPath.removePrefix(webjarsPrefix)
-            try {
-                val location = extractWebJar(resourcePath)
-                val stream = Webjars::class.java.classLoader.getResourceAsStream(location) ?: return
-                context.call.respond(
-                    InputStreamContent(
-                        stream,
-                        ContentType.defaultForFilePath(fullPath),
-                        lastModified
-                    )
-                )
-            } catch (multipleFiles: MultipleMatchesException) {
-                context.call.respond(HttpStatusCode.InternalServerError)
-            } catch (notFound: IllegalArgumentException) {
             }
         }
-    }
 
     /**
-     * Plugin installation companion.
+     * Makes no effect. Will be dropped in future releases.
      */
-    public companion object Plugin : ApplicationPlugin<ApplicationCallPipeline, Configuration, Webjars> {
-
-        override val key: AttributeKey<Webjars> = AttributeKey("Webjars")
+    @Suppress("unused")
+    @Deprecated("This is no longer used and will be dropped in future releases.", level = DeprecationLevel.ERROR)
+    public var zone: ZoneId = ZoneId.systemDefault()
+}
 
-        override fun install(
-            pipeline: ApplicationCallPipeline,
-            configure: Configuration.() -> Unit
-        ): Webjars {
-            val configuration = Configuration().apply(configure)
+private fun extractWebJar(path: String, knownWebJars: Set<String>, locator: WebJarAssetLocator): String {
```

Comment:
consider extracting public API to the separate file to make it easier to inspect